### ISSUE TITLE
negative loss while training on kings college dataset

### ISSUE THREAD

**Issue by rlrahulkanojia** [youngguncho/PoseNet-Pytorch#6]
20th train Loss: total loss -1.273 / pos loss 1.371 / ori loss 0.018
21th train Loss: total loss -1.584 / pos loss 1.084 / ori loss 0.016
22th train Loss: total loss -1.778 / pos loss 0.875 / ori loss 0.017
23th train Loss: total loss -1.884 / pos loss 0.764 / ori loss 0.018
24th train Loss: total loss -1.651 / pos loss 1.008 / ori loss 0.017
25th train Loss: total loss -1.831 / pos loss 0.801 / ori loss 0.018
26th train Loss: total loss -1.562 / pos loss 1.067 / ori loss 0.018
27th train Loss: total loss -1.713 / pos loss 0.965 / ori loss 0.016
28th train Loss: total loss -1.338 / pos loss 1.275 / ori loss 0.019
29th train Loss: total loss -1.722 / pos loss 0.964 / ori loss 0.016
30th train Loss: total loss -1.551 / pos loss 1.094 / ori loss 0.018
31th train Loss: total loss -1.603 / pos loss 1.084 / ori loss 0.016
32th train Loss: total loss -1.714 / pos loss 0.947 / ori loss 0.017
33th train Loss: total loss -1.465 / pos loss 1.029 / ori loss 0.025
34th train Loss: total loss -1.740 / pos loss 0.927 / ori loss 0.017
35th train Loss: total loss -1.579 / pos loss 0.987 / ori loss 0.022
36th train Loss: total loss -1.537 / pos loss 1.074 / ori loss 0.019
37th train Loss: total loss -1.515 / pos loss 1.123 / ori loss 0.018
38th train Loss: total loss -1.181 / pos loss 1.409 / ori loss 0.020
0th val Loss: total loss -2.384 / pos loss 0.434 / ori loss 0.009
1th val Loss: total loss -2.452 / pos loss 0.358 / ori loss 0.009
2th val Loss: total loss -2.407 / pos loss 0.375 / ori loss 0.011
3th val Loss: total loss -2.356 / pos loss 0.417 / ori loss 0.011
Train and Validaion error -1.696101188659668 / -2.395916223526001
========================================
========================================
Epoch 225/399
--------------------
0th train Loss: total loss -1.572 / pos loss 1.093 / ori loss 0.017
1th train Loss: total loss -1.546 / pos loss 1.108 / ori loss 0.017
2th train Loss: total loss -1.847 / pos loss 0.873 / ori loss 0.014
3th train Loss: total loss -1.953 / pos loss 0.679 / ori loss 0.018
4th train Loss: total loss -1.753 / pos loss 0.908 / ori loss 0.017
5th train Loss: total loss -1.600 / pos loss 1.083 / ori loss 0.016
6th train Loss: total loss -1.617 / pos loss 1.014 / ori loss 0.018
7th train Loss: total loss -1.760 / pos loss 0.888 / ori loss 0.018
8th train Loss: total loss -1.264 / pos loss 1.346 / ori loss 0.019
9th train Loss: total loss -1.606 / pos loss 1.040 / ori loss 0.018
10th train Loss: total loss -1.619 / pos loss 0.982 / ori loss 0.020
11th train Loss: total loss -1.708 / pos loss 0.935 / ori loss 0.018
12th train Loss: total loss -1.645 / pos loss 1.058 / ori loss 0.015
13th train Loss: total loss -1.703 / pos loss 0.959 / ori loss 0.017
14th train Loss: total loss -1.768 / pos loss 0.860 / ori loss 0.019
15th train Loss: total loss -1.705 / pos loss 0.931 / ori loss 0.018
16th train Loss: total loss -1.712 / pos loss 0.958 / ori loss 0.016
17th train Loss: total loss -1.656 / pos loss 0.974 / ori loss 0.018
18th train Loss: total loss -1.686 / pos loss 0.984 / ori loss 0.016
19th train Loss: total loss -1.657 / pos loss 0.989 / ori loss 0.018

pos out [ 60.13612   -37.06963     1.4968748]
ori_out [ 1.43032163  0.86646476 -0.1398881 ]
pos true [ 58.309566 -36.34202    1.250287]
ori true [ 1.38532974  0.91265223 -0.18131388]
[ 60.13612   -37.06963     1.4968748]
[ 58.309566 -36.34202    1.250287]
1219th Error: pos error 1.982 / ori error 0.077
====================
Overall median pose errer 0.973 / 0.054
Overall average pose errer 1.162 / 0.069

**Comment by LINFF1023** [comment]
Is this normal?

**Comment by LINFF1023** [comment]
This is normal do？
